### DISCUSSION TITLE
Need to configure AWS cli

### DISCUSSION THREAD

**Discussion by Debashis-Dey-IAG** [actions/runner#2057] [General]
We are using github action pipeline with Selfhosted runner latest version 2.295 . Need to configure aws cli in workflow . can you please guide me or if any version is available with aws cli

**Comment by Jamesking56** [comment]
AWS have published an official GitHub Action that let's you configure the CLI: https://github.com/marketplace/actions/configure-aws-credentials-action-for-github-actions
